### ISSUE TITLE
VideoClientSDK.h is missing nullability type specifiers

### ISSUE THREAD

**Issue by sjmadsen** [aws/amazon-chime-sdk-ios#83]
**Describe the bug**
`VideoClientSDK.h` in the latest AmazonChimeSDKMedia framework is missing nullability type specifiers throughout the file:

```<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "Headers/AmazonChimeSDKMedia.h"
        ^
/Users/.../AmazonChimeSDKMedia.framework/Headers/AmazonChimeSDKMedia.h:17:9: note: in file included from /Users/.../AmazonChimeSDKMedia.framework/Headers/AmazonChimeSDKMedia.h:17:
#import "VideoClientSDK.h"
        ^
/Users/.../AmazonChimeSDKMedia.framework/Headers/VideoClientSDK.h:23:46: error: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)
@property (atomic, strong, readonly) NSString* identifier;
                                             ^
/Users/.../AmazonChimeSDKMedia.framework/Headers/VideoClientSDK.h:23:46: note: insert '_Nullable' if the pointer may be null
@property (atomic, strong, readonly) NSString* identifier;
                                             ^
/Users/.../AmazonChimeSDKMedia.framework/Headers/VideoClientSDK.h:23:46: note: insert '_Nonnull' if the pointer should never be null
@property (atomic, strong, readonly) NSString* identifier;
                                             ^
<unknown>:0: error: could not build Objective-C module 'AmazonChimeSDKMedia'
```

**To Reproduce**
1. Build a Swift app with ChimeSDKMedia 0.6.0.

**Expected behavior**
No compiler errors in framework headers.

**Test environment Info (please complete the following information):**
 - Device: n/a
 - OS: n/a
 - Xcode 11.6
 - Version AmazonChimeSDK: 0.8.1
 - Version AmazonChimeSDKMedia: 0.6.0

**Comment by hokyungh** [comment]
Thank you for reporting issue. It seems we haven't changed our `VideoClientSDK.h`. Would you be able to clean build once more, so 
1. Close xcode
2. `cd ~/Library/Developer/Xcode/DerivedData` and delete the temp project folder
3. Rebuild the project

Let us know if this still an issue

**Comment by sjmadsen** [comment]
Definitely still a problem. Perhaps something else changed that now uses `VideoClientSDK.h` in such a way that Swift sees it and is unhappy about the missing type specifiers?

**Comment by hokyungh** [comment]
Sorry there was a change in `VideoClientSDK.h`, but it wasn't the line where it errors. I can't seem to reproduce in Xcode 11.2.1. or 11.3, but one of my colleagues with 11.6 see it as warning.
For now, would you be able to try to set [Nullable flag](https://stackoverflow.com/questions/32539285/pointer-is-missing-a-nullability-type-specifier)
```
1. Go to Build Settings
2. Make sure you are viewing the "All" tab not "Basic" or "Customized"
3. Search for the field Other Warning Flags.
4. Add the following flag: -Wno-nullability-completeness.
5. Now clean and build (⇧⌘K then ⌘R)
6. Quit Xcode and re-open it (Don't skip this step!)
```

We'll continue to investigate on this matter. Let us know above workaround works.

**Comment by hokyungh** [comment]
@sjmadsen Thank you for your patience. This issue should be fixed in 0.9.0 and 0.6.2.

**Comment by alnlau** [comment]
Hello @sjmadsen ,

We've released SDK 0.9.0 and SDK Media 0.6.2. Let us know if you are still encountering this issue after upgrading.
